### PR TITLE
Escape Generic Information

### PR DESCRIPTION
**Pull request by tobwiens**
Escape Generic Information

### CODE REVIEW

File: app/scripts/proactive/templates/job-template.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <% if (job["Generic Info"] && job["Generic Info"].length>0) { %>
     <genericInformation>
     <% _.each(job["Generic Info"], function(gi) { %>
-            <info name="<%=gi["Property Name"]%>" value="<%=gi["Property Value"]%>"/>
+            <info name="<%=gi["Property Name"]%>" value="<%= _.escape(gi["Property Value"]) %> "/>
```
Comment (marcocast) [inline app/scripts/proactive/templates/job-template.html]: Why we are escaping gi?

Comment (tobwiens) [inline app/scripts/proactive/templates/job-template.html]: We put a json object inside the GI. the EXECUTION_CALENDARS object. That introduces thinks like < {} which will be interpreted as XML. That breaks the parsing of the workflow inside the Execution Scheduler

Comment (tobwiens) [inline app/scripts/proactive/templates/job-template.html]: I tested Scheduler submission as well. It fails for the same reason:

Invalid workflow
At [tag:job] : Element type "info" must be followed by either attribute specifications, ">" or "/>".

Comment (MaelAudren) [inline app/scripts/proactive/templates/job-template.html]: I think this an acceptable solution however we should be aware that if we want to validate the gi values as we do for variables, we will have to find another solution.

**Comment by fviale** [comment]
The escape function must also be used in the Task Generic information definition.

**Pull request merged**